client/src/components/Post/PostAuthor/PostAuthor.jsx

```diff
@@ -1,21 +1,50 @@
 import React, { Fragment } from 'react'
 import useStyles from './postAuthorStyles'
 import Avatar from '@material-ui/core/Avatar'
+import ArrowRightIcon from '@material-ui/icons/ArrowRight'
+import PropTypes from 'prop-types'
 
-const PostAuthor = ( props ) => {
+const PostAuthor = ( { author, owner, date} ) => {
   const classes = useStyles()
 
+  const postDate = (dateMilliseconds) => {
```

Comment:
We'll have date in other components as well (comments, at least) my suggestion is to extract this function to some separate utils file